lib/prometheus_exporter/server/puma_collector.rb

```diff
@@ -11,6 +11,8 @@ class PumaCollector < TypeCollector
       request_backlog: "Number of requests waiting to be processed by a puma thread.",
       thread_pool_capacity: "Number of puma threads available at current scale.",
       max_threads: "Number of puma threads at available at max scale.",
+      busy_threads:
```

Comment:
Sorry one more question, should we check for the puma version before exposing this metric? Otherwise, this metric  will always be 0 on puma versions that don't support the `busy_threads`.

Reply:
Ah, I think it's a good idea. I will make that change.